### DISCUSSION TITLE
[DOC]:Plugin Document Specification

### DISCUSSION THREAD

**Discussion by Takuka0311** [alibaba/ilogtail#253] [Ideas]
#224 感觉文档可以增加一些统一的规范，方便阅读和编辑。这里是一些建议：
1、所有插件文档的命名：以插件英文名命名，“_”改为“-”，例如“metric_mock”插件的文档明名为“metric-mock.md”
2、插件文档格式
2.1、简介：最后统一添加源代码链接。
2.2、配置参数：三列，其中“类型，默认值”处，类型统一按Java类型写，默认值处特殊的有，空字符串：`""`，空数组：`[]`，空map：`{}`。
2.3、样例：代码块加上代码语言，如yaml
3、在data-pipeline的overview.md里，所有的插件按英文名字典序升序排列，添加的时候注意插入的位置
4、在SUMMARY.md里，所有的插件顺序与overview.md保持一致

**Comment by messixukejia** [comment]
很好的建议，后续新的文档按照这个模式来。存量的逐步修改掉。

需要有一个标准的PR，加到贡献文档中，供后其他贡献者参考

**Reply by Takuka0311** [reply]
好的，标准文档后续就添加

**Reply by Takuka0311** [reply]
文档标准： https://open.observability.cn/project/loongcollector/plugin-doc-templete/#_top
